packages/cli/src/commands/prisma.js

```diff
@@ -12,122 +12,103 @@ export const command = 'prisma [commands..]'
 export const description = 'Run Prisma CLI with experimental features'
 
 /**
- * This is a lightweight wrapper around Prisma's CLI.
- *
- * In order to test this command you can do the following:
- * 0. cd __fixtures__/example-todo-main && yarn install
- * 1. cd..; yarn build:watch
- * 2. cd packages/cli
- * 3. RWJS_CWD=../../__fixtures__/example-todo-main yarn node dist/index.js prisma <test commands>
+ * This is a lightweight wrapper around Prisma's CLI with some Redwood CLI modifications.
  */
-export const builder = async (yargs) => {
-  // accept either help or --help, which is the same behavior as all the other RW Yargs commands.
-  const argv = mapHelpCommandToFlag(process.argv.slice(3))
-
-  // We dynamically create the `--options` that are passed to this command.
-  // TODO: Figure out if there's a way to turn off yargs parsing.
-  const options = argv
-    .filter((x) => x.startsWith('--'))
-    .map((x) => x.substr(2))
-    .reduce((pv, cv) => {
-      return {
-        ...pv,
-        [cv]: {},
-      }
-    }, {})
-
+export const builder = (yargs) => {
+  // Disable yargs parsing of commands and options because it's forwarded
+  // to Prisma CLI.
   yargs
+    .strictOptions(false)
+    .strictCommands(false)
+    .strict(false)
+    .parserConfiguration({
+      'camel-case-expansion': false,
+    })
     .help(false)
     .version(false)
-    .option(options)
-    .option('version', { alias: 'v' })
-
-  const paths = getPaths()
-
-  const autoFlags = []
+}
 
-  const hasHelpFlag = argv.some(
-    (arg) => arg.includes('--help') || arg.includes('-h')
-  )
+// eslint-disable-next-line no-unused-vars
+export const handler = async ({ _, $0, commands = [], ...options }) => {
+  const rwjsPaths = getPaths()
 
-  // Only pass auto flags, when not running help
-  if (!hasHelpFlag) {
-    if (['seed'].includes(argv[1])) {
-      // this is safe as is if a user also adds --preview-feature
-      autoFlags.push('--preview-feature')
-    }
+  // Prisma only supports '--help', but Redwood CLI supports `prisma <command> help`
+  const helpIndex = commands.indexOf('help')
+  if (helpIndex !== -1) {
+    options.help = true
+    commands.splice(helpIndex, 1)
+  }
 
+  // Automatically inject options for some commands.
+  const hasHelpOption = options.help || options.h
+  if (!hasHelpOption) {
     if (
-      // `introspect` to be replaced by `db pull`; still valid as of prisma@2.19
       ['generate', 'introspect', 'db', 'migrate', 'studio', 'format'].includes(
-        argv[0]
+        commands[0]
       )
     ) {
-      if (!fs.existsSync(paths.api.dbSchema)) {
-        console.error(
-          c.error('\n Cannot run command. No Prisma Schema found.\n')
-        )
+      if (!fs.existsSync(rwjsPaths.api.dbSchema)) {
+        console.error()
+        console.error(c.error('No Prisma Schema found.'))
+        console.error(`Snaplet searched here '${rwjsPaths.api.dbSchema}'`)
```

Comment:
We should remove "Snaplet" from this copy".

Reply:
Lol, thanks for finding that. It was a long night.